### PR TITLE
$_WD_CONSOLE_Sufix moved @CRLF to internall scope

### PR DESCRIPTION
**Pull request by mlipok**
### Proposed changes

There is not needed in all __WD_ConsoleWrite line to add `@CRLF  ` 
It should be moved to to internall processing into `__WD_ConsoleWrite()`  function.

A the same time:
`__WD_ConsoleWrite` should support for other suffixes not olny `@CRLF`
For example If I use:
```autoit
_WD_Option("console", _Log_To_DataBase)
```
Then there is no need to write `@CRLF` to data base which stores log entries in separate database rows.

So using
```autoit
_WD_Option("consolesuffix", "")
_WD_Option("console", _Log_To_DataBase)
```
will fit this needs.

This is also code refactoring as now we ara not using `@CRLF` all the time when `__WD_ConsoleWrite()` is used.

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [ ] Bugfix (change which fixes an issue)
- [x] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

Need to add `@CRLF` in all `__WD_ConsoleWrite()` calls.
No way to change behavior in case you want to store logs suffix in database.

### What is the new behavior?
There is no need to add `@CRLF` in all `__WD_ConsoleWrite()` calls.
Easy way to change behavior/suffix in case you want to store logs in database.

### Additional context

https://github.com/Danp2/au3WebDriver/issues/219

### System under test

Please complete the following information.
NOT RELATED

### CODE REVIEW

File: wd_core.au3 [hunk 1]
```diff
@@ -174,6 +174,7 @@ Global $_WD_RESPONSE_TRIM = 100 ; Trim response string to given value for debug
 Global $_WD_ERROR_MSGBOX = True ; Shows in compiled scripts error messages in msgboxes
 Global $_WD_DEBUG = $_WD_DEBUG_Info ; Trace to console and show web driver app
 Global $_WD_CONSOLE = ConsoleWrite ; Destination for console output
+Global $_WD_CONSOLE_Sufix = @CRLF ; Suffix added to the end of Message in $_WD_CONSOLE function
```
Comment (Danp2) [inline wd_core.au3:177]: There should be two `f`s in suffix

File: wd_core.au3 [hunk 2]
```diff
@@ -1256,25 +1261,25 @@ Func _WD_Startup()
 			$sWinHttpVer &= " (Download latest source at <https://raw.githubusercontent.com/dragana-r/autoit-winhttp/master/WinHttp.au3>)"
 		EndIf
 
-		_WinAPI_GetBinaryType($_WD_DRIVER)
+		_WinAPI_GetBinaryType(@ScriptDir & "\" & $_WD_DRIVER)
```
Comment (Danp2) [inline wd_core.au3:1264]: This change shouldn't occur as we removed this earlier today

File: wd_core.au3 [hunk 3]
```diff
@@ -1134,6 +1135,10 @@ Func _WD_Option($sOption, $vValue = Default)
 			EndIf
 			$_WD_CONSOLE = $vValue
 
+		Case "consolesufix"
```
Comment (Danp2) [inline wd_core.au3:1138]: Same spelling issue here. Also, need to add this option in function header.

**Review by Danp2 — CHANGES_REQUESTED** [review]

File: wd_core.au3 (hunk 2 above)
Comment (mlipok) [reply wd_core.au3:1264]: Ye... I gett v7 without fixes....

File: wd_core.au3 (hunk 3 above)
Comment (mlipok) [reply wd_core.au3]: Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)

File: wd_core.au3 (hunk 2 above)
Comment (mlipok) [reply wd_core.au3]: Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)

File: wd_core.au3 (hunk 1 above)
Comment (mlipok) [reply wd_core.au3]: Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)

File: wd_core.au3 (hunk 3 above)
Comment (mlipok) [reply wd_core.au3]: > Also, need to add this option in function header.

And here: [suplemented](https://github.com/Danp2/au3WebDriver/pull/242/commits/3221c678875aa98bd09cd885de21c05a27bb5eee)

**Review by Danp2 — APPROVED** [review]

**Pull request merged**